tests/foreman/ui/test_user.py

```diff
@@ -1057,6 +1059,71 @@ def test_positive_mails_for_new_timezone(self):
         @CaseLevel: Integration
         """
 
+    @stubbed()
+    @tier1
+    def test_positive_tab_visible_by_user_with_edit_params_permission(self):
```

Comment:
Since user will have access of parameters tab so can we re-name the test like: `test_positive_parameters_tab_access_with_edit_params` ?